### ISSUE TITLE
[Accessibility] Add tooltip to `MinimalFooter` icons

### ISSUE THREAD

**Issue by simurai** [primer/brand#353]
There was a recent [accessibility audit](https://github.com/github/accessibility-audits/issues/4955) for https://githubuniverse.com.

> Expected result
> On the footer section, the tooltips should be defined for the twitter, github, linkedin, youtube and facebook controls.

<img width="956" alt="Tool-tip are not defined for the twitter, github, linkedin, youtube and facebook controls" src="https://private-user-images.githubusercontent.com/73885337/253275388-d83b6459-e681-469f-bcd3-139f2f3e898c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE2ODk3MjY5NDMsIm5iZiI6MTY4OTcyNjY0MywicGF0aCI6Ii83Mzg4NTMzNy8yNTMyNzUzODgtZDgzYjY0NTktZTY4MS00NjlmLWJjZDMtMTM5ZjJmM2U4OThjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzA3MTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMwNzE5VDAwMzA0M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWRiMWU2ODdmYjNmMjhlYWUwOTljMzdlN2E1NjM0NWI0ZjMwNjExMjQ2YmU2N2UxZDA3MmY1YWY5ZjQyN2ViMjImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.-XECK5gxSHO-_NjsqTuJcgEDCeU6FkIf8nuH0xxZyhA" style="max-width: 100%;">

The `MinimalFooter` component currently has the `visually-hidden` text for screen readers. But I assume we would also need something for mouse and keyboard users that might not know what the icon (e.g. twitter bird) means. In the audit it says adding a "tooltip", but not sure if it needs to be a [Tooltip](https://primer.style/react/Tooltip) like we have in Primer React? It's still marked as "Not reviewed for accessibility". Or if adding a `title` attribute that the browser shows as a tooltip would be enough? Maybe something to get clarity with the `#accessibility` team first.

**Comment by joseph-lozano** [comment]
Discussed with a11y working group at the Accessibility Eng+ Office Hour. This is no longer considered a WCAG failure. We will follow up with this when Primer implements its Tooltip component (currently deferred to Q2).

**Comment by joshfarrant** [comment]
Closing this as it's not been raised in any recent audits. If it's still relevant, feel free to re-open.
